### PR TITLE
Fetch Twilio SMS Message Example

### PR DESCRIPTION
**Pull request by twistedpair**
My first contribution. Pointers welcome.

Adds an example of fetching the latest inbound SMS matching a number via a Twilio account, consuming the user's Twilio credentials via a mabl Credential.

### CODE REVIEW

File: mabl snippets/getLatestTwilioSms.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ *                              the javascript step and provide a value to the following
+ *                              steps of the flow/journey. A return statement from this
+ *                              function call will not provide any results for use
+ *                              in the following steps in this flow or journey.
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  // Add Twilio API creds as username: Account SID, password: auth token, add to the plan
+  var twilioAccountSid = mablInputs.variables.web.defaults.credentials.username;
+  var twilioAccountAuthToken = mablInputs.variables.web.defaults.credentials.password;
```
Comment (DancingNerd) [inline mabl snippets/getLatestTwilioSms.js:26]: Tests can only ever use 1 set of credentials per run. If a test uses our Credentials feature to login then they would not be able to use them here. Might make sense to outline these as environment variables or just name the variable here instead.

**Review by DancingNerd — APPROVED** [review]
This all looks good and makes sense to me! 

The only concern I have is what happens if it takes more than 30 seconds to receive the SMS? Our JS steps timeout after 30 seconds and the test would fail? 
You know more about the Twillo system, could it take longer?

**Comment by twistedpair** [comment]
> The only concern I have is what happens if it takes more than 30 seconds to receive the SMS? Our JS steps timeout after 30 seconds and the test would fail?
You know more about the Twillo system, could it take longer?

Good point. That would trigger the aborted error message.

SMS should be near instant, but the customer (sending) system could be slow. I think using a fixed wait before calling this Javascript Snippet would be the approach for that case.

File: mabl snippets/getLatestTwilioSms.js (hunk 1 above)
Comment (twistedpair) [reply mabl snippets/getLatestTwilioSms.js:26]: Configs broken out with alt example.

**Pull request merged**